schemas/secret-list.yml

```diff
@@ -0,0 +1,18 @@
+id: http://schemas.taskcluster.net/secrets/v1/secret-list.json#
+$schema:  http://json-schema.org/draft-04/schema#
+title:        "A list of TaskCluster Secrets"
```

Comment:
Unfortunately this results in a go type [AListOfTaskClusterSecrets](https://github.com/taskcluster/taskcluster-client-go/compare/1dfc559d9b23...ccb2188c159f#diff-f6fa27944cef0ebaeffad07bbf08de8fR168) being generated - I'll raise a separate PR to resolve this.